proposals/interface-types/working-notes/scenarios/unicode_count.md

```diff
@@ -133,17 +132,17 @@ it:
   (param $str string) (result u32)
   local.get $str
   string.size
-  call $malloc
-  deferred (i32)
-    let (local $tgt i32)
-      local.get $str
-      local.get $tgt
-      string-to-memory "memx"
-      call $countCodes_impl
-      i32-to-u32
-    end
-  finally
-    call $free
+  call-export "ex:malloc"
+  own (i32)
+    call-export "ex:free"
+  end
+  let (result u32) (local $tgt (owned i32))
+    local.get $str
+    local.get $tgt
+	owned.access
```

Comment:
Weird indentation here

Reply:
caused by emacs. Do not always remember to set untabify :(